app.py

```diff
@@ -40,6 +43,15 @@
 waf_stack = WAFStack(app, config, ocs_services.load_balancer_arn)
 waf_stack.add_dependency(ocs_services)
 
+guardduty_stack = GuardDutyStack(app, config)
+guardduty_stack.add_dependency(ocs_services)
+
+securityhub_stack = SecurityHubStack(app, config)
+securityhub_stack.add_dependency(ocs_services)
+
+DetectiveStack = DetectiveStack(app, config)
+DetectiveStack.add_dependency(ocs_services)
```

Comment:
nit: consistent naming:
```suggestion
detective_stack = DetectiveStack(app, config)
detective_stack.add_dependency(ocs_services)
```